lib/iris/fileformats/netcdf.py

```diff
@@ -780,6 +784,13 @@ def write(self, cube, local_keys=None, unlimited_dimensions=None,
                     'cf_patch')
                 warnings.warn(msg)
 
+        # Perform any user registered callback function.
+        # As the NetCDF saver commits to disk as it processes the cube and
+        # its metadata, it is not possible to skip or ignore the derived
+        # NetCDF representation of a cube as with other file format savers.
```

Comment:
What do you think of bjlittle/iris#12? What would your user requests look like if they used something like that? How would you feel about explaining it in a support response?